src/Utilities/constants.js

```diff
@@ -268,3 +275,36 @@ export const hybridInventoryTabKeys = {
     url: '/manage-edge-inventory',
   },
 };
+
+export const LOADING_BIFROST_TABLE = [
```

Comment:
This works just fine. However, I think we can also use the SkeletonTable from fec-components package as it was used here: https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/src/components/InventoryTable/EntityTable.js#L11C32-L11C88.